components/Navbar.vue

```diff
@@ -168,6 +168,10 @@ export default class NavbarMenu extends mixins(PrefixMixin) {
     return this.urlPrefix === 'rmrk' || this.urlPrefix === 'westend'
   }
 
+  get isBsx(): boolean {
```

Comment:
above in the code there is a check `   v-if="urlPrefix === 'bsx'"`
could you replace this with `isBsx`

Reply:
oki